contracts/DataStructures/PriorityQueue.sol

```diff
@@ -0,0 +1,107 @@
+pragma solidity ^0.4.0;
+
+import 'zeppelin-solidity/contracts/math/SafeMath.sol';
+
+/**
+ * @title PriorityQueue
+ * @dev A priority queue implementation
+ */
+contract PriorityQueue {
+  using SafeMath for uint256;
+
+  /*
+   *  Modifiers
+   */
+  modifier onlyOwner() {
```

Comment:
Zeppelin has an Ownable base library you can use

Reply:
Ah NVM, I see this is an open source library. In that case, could it be included as a dependency?